rdsbroker/broker.go

```diff
@@ -34,16 +34,30 @@ var (
 )
 
 var rdsStatus2State = map[string]brokerapi.LastOperationState{
-	"available":                    brokerapi.Succeeded,
-	"backing-up":                   brokerapi.InProgress,
-	"creating":                     brokerapi.InProgress,
-	"deleting":                     brokerapi.InProgress,
-	"maintenance":                  brokerapi.InProgress,
-	"modifying":                    brokerapi.InProgress,
-	"rebooting":                    brokerapi.InProgress,
-	"renaming":                     brokerapi.InProgress,
-	"resetting-master-credentials": brokerapi.InProgress,
-	"upgrading":                    brokerapi.InProgress,
+	"available":                           brokerapi.Succeeded,
+	"backing-up":                          brokerapi.InProgress,
+	"creating":                            brokerapi.InProgress,
+	"deleting":                            brokerapi.InProgress,
+	"maintenance":                         brokerapi.InProgress,
+	"modifying":                           brokerapi.InProgress,
+	"rebooting":                           brokerapi.InProgress,
+	"renaming":                            brokerapi.InProgress,
+	"resetting-master-credentials":        brokerapi.InProgress,
+	"upgrading":                           brokerapi.InProgress,
+	"configuring-enhanced-monitoring":     brokerapi.InProgress,
+	"starting":                            brokerapi.InProgress,
+	"stopping":                            brokerapi.InProgress,
+	"stopped":                             brokerapi.InProgress,
+	"storage-full":                        brokerapi.InProgress,
```

Comment:
`storage-full` being in-progress might lead to some operations never being marked as succeeded, which would prevent a tenant from scaling their instance up. But we have monitors to alert support if a tenant is running out of storage, and thus I think this is okay.

Reply:
@46bit In most of the real world scenarios it shouldn't cause an issue. I imagine it will cause one if the storage is full but the tenant doesn't scale it up in an update (but does something different). But if the tenant upscales the instance the storage-full state will eventually go away and LastOperation will return with success.